install_config/downgrade.adoc

```diff
@@ -38,7 +38,7 @@ etcd data directory. Ensure these exist on your masters and etcd members:
 
 ----
 /etc/origin/master/master-config.yaml.<timestamp>
-/var/lib/etcd/openshift-backup-<timestamp>
+/var/lib/etcd/openshift-backup-pre-upgrade-20180207123542 
```

Comment:
/var/lib/etcd/openshift-backup-pre-upgrade-<timestamp>
looks more accurate and style consistent